src/test/resources/data/typicalDataBundle.json

```diff
@@ -750,35 +759,51 @@
     }
   },
   "feedbackSessions": {
+    "sessionInTestingGiverType": {
+      "feedbackSessionName": "Testing giver type session",
+      "courseId": "idOfSessionInTestingGiverType",
```

Comment:
It seems like this course doesn't exist in the bundle. Can this be problematic?

Reply:
For example: perhaps someone adds a check to the `logic` class that tests that the course actually exists.